pages/career-quiz.tsx

```diff
@@ -27,6 +27,13 @@ const CareerQuiz = () => {
     setStage((prevStage) => prevStage - 1);
   };
 
+  // Define the object and initialize it with empty values
+  const [quizObject, setQuizObject] = useState({
+    industries: [],
```

Comment:
This is fine here, but I think we don't have to specify this type in the shared component with these exact keys.

In the shared component you can use the Record type in typescript.

Search for other uses of the Record type in our repo